.devops/pagopa-code-review-pipelines.yml

```
@@ -34,7 +34,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-2800-calculate-fees-multi
```

```suggestion
      ref: master
```
TODO: to be restored after dependent PR merge